Sources/MapboxDirections/LaneIndication.swift

```diff
@@ -69,6 +69,10 @@ public struct LaneIndication: OptionSet, CustomStringConvertible {
         self.init(rawValue: laneIndication.rawValue)
     }
     
+    public init?(from direction: ManeuverDirection) {
+        self.init(descriptions: [direction.rawValue])
```

Comment:
Add a comment noting the assumption that every possible raw value of ManeuverDirection matches a valid raw value of LaneIndication.